### PR TITLE
Sales Accounts use Https for SecurityProtocol

### PR DESCRIPTION
**Pull request by RossStewart**

### CODE REVIEW

**Comment by lewisrenfrew** [comment]
Won't you need to change some urls in configs too? 

Do https calls work when running locally?

**Comment by lewisrenfrew** [comment]
> > Won't you need to change some urls in configs too?
> > Do https calls work when running locally?
> 
> I was wondering this too to be honest. I noticed when Andrew did this for Sales Orders there was a change to a Web.Config file but can't see any similar files here - is this stored somewhere else?

when running locally it'll be your config.env
when deployed to sys / prod it'll be sys.env and production.env in the relevant s3 bucket

File: src/Proxy/DiscountingService.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public DiscountScheme GetDiscountScheme(string discountSchemeUri)
 
             var uri = new Uri($"{this.proxyRoot}{discountSchemeUri}", UriKind.RelativeOrAbsolute);
 
+            ServicePointManager.SecurityProtocol = (SecurityProtocolType)3072;
```
Comment (richardiphillips) [inline src/Proxy/DiscountingService.cs:38]: i'd change the paths in the s3 config before doing this. it's a more up to date restclient so *might* not need this

Comment (richardiphillips) [reply src/Proxy/DiscountingService.cs:38]: or might be easier to fix in the common library

Comment (RossStewart) [reply src/Proxy/DiscountingService.cs:38]: OK, I'll give it a go. Just sorting some other issues first.

**Pull request closed without merging**